AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DeckPickerContextMenu.java

```diff
@@ -122,7 +130,8 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         Collection col = CollectionHelper.getInstance().getCol(getContext());
         long did = getArguments().getLong("did");
         boolean dyn = col.getDecks().isDyn(did);
-        ArrayList<Integer> itemIds = new ArrayList<>(9);
+        ArrayList<Integer> itemIds = new ArrayList<>();
```

Comment:
```suggestion
        ArrayList<Integer> itemIds = new ArrayList<>(10); // init with our fixed list size for performance
```

@Arthur-Milchior went to quite a bit of trouble to pre-size all of the arrays in the system to gain a bit of performance, keeping the initializer here and just increasing it one is optimal I believe

Reply:
This has been updated.